multimodal/src/autogluon/multimodal/learners/base.py

```diff
@@ -193,7 +190,7 @@ def __init__(
             - 'image_text_similarity': Text-image similarity problem
             - 'feature_extraction': Extracting feature (only support inference)
             - 'zero_shot_image_classification': Zero-shot image classification (only support inference)
-            - 'few_shot_text_classification': (experimental) Few-shot text classification
+            - 'few_shot_classification': Few-shot classification for image or text data.
```

Comment:
[minor] does this work for matching and document classification?

Reply:
Not tested for document data yet.

Reply:
Doesn't support matching.